package.json

```diff
@@ -17,11 +17,14 @@
   "contributors": [],
   "license": "OGL-UK-3.0",
   "dependencies": {
+    "@azure/identity": "^3.0.1",
```

Comment:
Would suggest not using `^` before versions.  That way can ensure consistent package usage on every build

Reply:
agreed, and removed as suggested